jenkins/docker/docker-re-release.jenkinsfile

```diff
@@ -17,6 +17,14 @@ pipeline {
         timeout(time: 2, unit: 'HOURS')
     }
     agent none
+    triggers {
+        parameterizedCron '''
+            H 19 15 * * %PRODUCT=opensearch;TAG=1
+            H 19 15 * * %PRODUCT=opensearch-dashboards;TAG=1
+            H 19 15 * * %PRODUCT=opensearch;TAG=2
+            H 19 15 * * %PRODUCT=opensearch-dashboards;TAG=2
+        '''
+    }
```

Comment:
Do we need to add an assertion for this? Just in case someone changes this behavior it should fail the groovy tests?

Reply:
Updated now, thanks!